README.md

```diff
@@ -15,14 +15,22 @@ Have fun 😃
 `/gen-build`: Generates up to 12 builds for your selected game version. *Pro Tip: You can also exclude inline-bikes if you don't like them 🙃*
 
 ## Building the bot by yourself
-If you want for whatever reason want to build the bot by yourself you can do so by pulling this repository and configure it with your bot token.
-Create a config.json inside the root folder of the project and fill it with the following.
+If, for whatever reason you want to build the bot by yourself, you can do so by pulling this repository and configuring it with your bot token. Your bot token can be found in the [Discord Developer Portal](https://discord.com/developers/applications) under your bot's name, select the "Bot" option under settings and then you should be able to see it underneath the bots username.
+
+If you want to build a brand new bot, you'll need to give the bot the "Bot" and "Applications.Commands" permissions under scope.
+Under "Bot Permissions", you'll need to enable "Read Messages/View Channels", "Send Messages", and "Embed Links".
+
+Once thats finished, go into your code and create a config.json file inside the root folder of the project and fill it with the following:
 ```JSON
 {
   "Token": "<your-bot-token-here>"
 }
 ```
 
+If you want to test the bot on a specific server, you will need to replace the default GUID with your discord server ID that you want to test it on. This can be found when you right click on the server icon on left server menu. *(You will need to have [Developer Mode enabled](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) for you to be able to see this id.)* 
+
+You can find the default GUID on line 11 in the "CommandRegisterer" class in the "Classes" folder.
+
```

Comment:
Isn't necessary as the `GUILD` in the code was for something way different. This was used to register completely new commands just for the server as otherwise they will be registered globally which can take up to 1 hour to appear on discord. However when the bot joins the test server for the first time the commands are registered there immediately and are available for use. the `GUILD` parameter will be removed with PR #12

Reply:
Ok, I'll remove this section